pkg/execution/svccatadmission/internaldns.go

```diff
@@ -0,0 +1,103 @@
+package svccatadmission
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/atlassian/voyager/pkg/microsserver"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/internaldns"
+	"github.com/atlassian/voyager/pkg/servicecentral"
+	sc_v1b1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/pkg/errors"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	"k8s.io/api/extensions/v1beta1"
```

Comment:
might be better to call this `ext_v1beta1` which is what we've tended to use for this package in the past. especially with the other `v1beta1` items already referenced (better to alias them all I think)